pages/api-reference/funding-sources/remove.mdx

```diff
@@ -9,7 +9,9 @@ meta:
 
 # Remove a funding source
 
-Remove a funding source by id. A removed funding source is soft deleted and can still be accessed when retrieved. When a funding source with an `unverified` status is removed, an attempt to re-add it within 48 hours of the initial created date will re-activate the removed funding source and maintain the same `id`.
+This section outlines how to remove a funding source, both bank account and `VAN`, by id. A removed funding source is soft deleted and can still be accessed when retrieved.
+
+When a bank funding source with an `unverified` status is removed, an attempt to re-add it within 48 hours of the initial created date will re-activate the removed funding source and maintain the same `id`. Removing and readding a `VAN` however, will create a different `id`.
```

Comment:
Correction: "Removing and re-adding a `VAN` however, will create a different `id`"